internal/cmd/marketplace/apply/table.go

```diff
@@ -44,7 +44,7 @@ func buildTable(headers []string, items []marketplace.ApplyResponseItem, columnT
 }
 
 func buildSuccessTable(items []marketplace.ApplyResponseItem) string {
-	headers := []string{"ID", "Item ID", "Status"}
+	headers := []string{"Object ID", "Item ID", "Status"}
```

Comment:
Object ID is an implementation detail, I'd not let the user be aware of it, I'd use a different term rather than the one used by MongoDB (what if we decide to change the unique id and then have to change the code here?)

Reply:
At the moment we have the notation `Object ID` in almost every command (even as the related flag name), we are opening this PR in order to align this command with the standard.

If we want to change the naming I would rather do it in a divided PR considering all the documentation that needs to be updated accordingly!

Reply:
p.s. : in the future our target is to remove `Object ID` replacing it with `Item ID` in order to remove any reference to the underlying data source

Reply:
>If we want to change the naming I would rather do it in a divided PR considering all the documentation that needs to be updated accordingly!

agreed

let's do this for other commands as well please @jac